### ISSUE TITLE
Git push updates all remote repos

### ISSUE THREAD

**Issue by ichik** [akonwi/git-plus#763]
[QUESTION]
-----------------
Git-Plus 8.1.0, Atom 1.30.0 x64, macOS High Sierra 10.13.6

Pushing from a repo with multiple configured remotes triggers push to all of them (checked only for two).
-----------------
Steps to reproduce:
1. Create Commit
2. Push from Git-Plus panel
3. No prompt to select remote, push just goes everywhere

**Comment by brycefranzen** [comment]
Agreed. This should have the OLD behavior.  Should ask where to push.  I miss it :/

**Comment by ichik** [comment]
Okay, now it sometimes displays the remote selection prompt. Upon the selection of remote push further pushes become unavailable. Git-Plus: Push doesn't trigger anything.

**Comment by alliter8** [comment]
This is still an issue today. Is there a solution? The documentation for Git Plus still says that it will prompt you if there are multiple repos; it does not.
